contrib/checkbox-provider-ce-oem/units/strict-confinement/mediacard-strict-confinement.pxu

```diff
@@ -0,0 +1,64 @@
+plugin: user-interact
+category_id: strict-confinement-mode
+id: strict-confine/mediacard/sdhc-insert
+flags: also-after-suspend
+estimated_duration: 30.0
+command:
+  test-strict-confinement.run-watcher insertion mediacard
+imports:
+  from com.canonical.plainbox import manifest
+  from com.canonical.certification import snap
+  from com.canonical.certification import lsb
+requires:
+  manifest.has_card_reader == 'True'
+  lsb.distributor_id == 'Ubuntu Core'
+  snap.name == 'test-strict-confinement'
+user: root
+_summary: Test that insertion of an SDHC card is detected
+_description:
+ PURPOSE:
```

Comment:
For the `PURPOSE`, `STEPS` and ` VERIFICATION` in the `_description` field, we could use `_purpose`, `_step_` and `_verification`.
You may refer to [here](https://checkbox.readthedocs.io/en/latest/tutorial/using-checkbox/advanced-commands.html#list-available-objects)

Reply:
Fixed and thanks for the review.
Please refer to the sideload result.
https://pastebin.canonical.com/p/KCgZMGhxzZ/